app/controllers/feature_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FeatureController < ApplicationController
+  before_action :validate_feature!
+
+  # Toggles boolean for session based feature flags
+  def toggle
```

Comment:
If that's easier, I'm all for it. Sounds kind of like a Cadillac! You want me to make a ticket for that option? Or just a general one and you can decide what is most efficient?